service/frontend/api/handler.go

```diff
@@ -1889,13 +1889,26 @@ func (wh *WorkflowHandler) StartWorkflowExecution(
 	if err != nil {
 		return nil, err
 	}
-	wh.GetLogger().Debug("Start workflow execution request domainID", tag.WorkflowDomainID(domainID))
 	historyRequest, err := common.CreateHistoryStartWorkflowRequest(
 		domainID, startRequest, time.Now(), wh.getPartitionConfig(ctx, domainName))
 	if err != nil {
 		return nil, err
 	}
 
+	// for debugging jitter workflow
+	// will be removed later
+	jitterStartSeconds := startRequest.GetJitterStartSeconds()
+	if startRequest.GetDomain() == "cadence-canary" && jitterStartSeconds > 0 {
+		wh.GetLogger().Debug("Start workflow execution request domainID",
+			tag.WorkflowDomainID(domainID),
+			tag.WorkflowID(startRequest.WorkflowID),
+			tag.Dynamic("JitterStartSeconds", jitterStartSeconds),
+			tag.Dynamic("firstDecisionTaskBackoffSeconds", historyRequest.GetFirstDecisionTaskBackoffSeconds()),
+		)
```

Comment:
is it possible to cover these in tests ?

Reply:
@bowenxia I wonder if you need to have this permanently or just for debugging.
If it's purely for debugging (I guess so since tags are too specific) then you can temporarily deploy your own branch.